cmd/common-main.go

```diff
@@ -51,7 +51,7 @@ func init() {
 	logger.RegisterError(config.FmtError)
 
 	rand.Seed(time.Now().UTC().UnixNano())
-	globalDNSCache = xhttp.NewDNSCache(3*time.Second, 10*time.Second)
+	globalDNSCache = xhttp.NewDNSCache(10*time.Second, 3*time.Second)
```

Comment:
I guess the server already retries when the DNS resolution times out, right ?

Reply:
Yes that has nothing to do with this change.